site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/service/SiteServiceImpl.java

```diff
@@ -44,22 +53,40 @@ public SiteServiceImpl(final EntityStore<Site, String> siteStore,
         }
         this.siteStore = siteStore;
         this.entityValidation = entityValidation;
+        initMaps(configuration);
         logger.info(Services.STARTUP.message());
     }
 
+    private void initMaps(final SiteConfiguration configuration) {
```

Comment:
Don't need initMaps(), all the work is done in the recursive addTypesToMap method which can be invoked directly from the constructor.

Reply:
Agree, removed the initMaps()